terraform/modules/stack/base/outputs.tf

```diff
@@ -65,11 +65,20 @@ output "rds_postgres_security_group" {
 
 /* RDS Bosh Instance */
 output "bosh_rds_url" {
```

Comment:
IMO that works better for resources we can safely delete and recreate. For database instances, we'd need to `terraform state mv` any time we want to move a `_curr` instance to `_prev`.